src/html/phetpreview_en-US.html

```diff
@@ -0,0 +1,502 @@
+<html>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/foundation/5.5.2/css/foundation.css">
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/foundation/5.5.2/js/foundation.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.3/modernizr.js"></script>
+    
+    <link rel="stylesheet" href="../../src/lib/foundation/css/foundation.min.css" />
+    <link rel="stylesheet" href="../../src/lib/foundation/css/foundation.css" />
+    <link rel="stylesheet" href="../../src/lib/foundation/css/foundation-icons/foundation-icons.css" />
+    <!-- CSS required for the Prefs Editor -->
+    <link rel="stylesheet" type="text/css" href="../../src/lib/infusion/src/framework/preferences/css/Enactors.css" />
+    <link rel="stylesheet" type="text/css" href="../../src/lib/infusion/src/framework/preferences/css/PrefsEditor.css" />
+    <link rel="stylesheet" type="text/css" href="../../src/css/style.css" />
+    
+	<script>
+	    var keys = [];
+	    var messages = [];
+	    var currentMsg;
+	    $(document).ready(function() {
+	        $( "a" ).addClass( "has-tip tip-top" );

```

Comment:
See above comments for standards on JS files. Only the smallest driver functions should be written within markup, with the bulk of the implementation in .js files conforming to the project's coding standards.